webapp/components/expression/expressionEditorPopupState.js

```diff
@@ -106,6 +116,7 @@ export const useExpressionEditorPopupState = (props) => {
   const nodeDefCurrent = nodeDefUuidCurrent ? Survey.getNodeDefByUuid(nodeDefUuidCurrent)(survey) : null
 
   const variables = ExpressionVariables.getVariables({ survey, nodeDefContext, nodeDefCurrent, mode, lang })
+  const variablesGroupedByParentUuid = ExpressionVariables.getVariablesGroupedByParentUuid({ variables, survey })
```

Comment:
why two set of variables? wouldnt passing a boolean prop to indicate whether the variables should appear grouped by parent entity be enough?